src-cpp/app/modules/main/wallet/accounts/model.h

```diff
@@ -38,18 +31,14 @@ class Model : public QAbstractListModel
     explicit Model(QObject* parent = nullptr);
     ~Model() = default;
 
-    QHash<int, QByteArray> roleNames() const;
-    virtual int rowCount(const QModelIndex&) const;
-    virtual QVariant data(const QModelIndex& index, int role) const;
+    QHash<int, QByteArray> roleNames() const override;
+    int rowCount(const QModelIndex&) const override;
+    QVariant data(const QModelIndex& index, int role) const override;
```

Comment:
minor: are you sure its not `final`?

Reply:
I think we can keep it as override, final makes sure no one else is able ton inherit form this class further
/